cmd/descheduler/app/server.go

```diff
@@ -111,6 +112,9 @@ func NewDeschedulerCommand(out io.Writer) *cobra.Command {
 }
 
 func Run(ctx context.Context, rs *options.DeschedulerServer) error {
+	// increase the fake watch channel so the dry-run mode can be run
+	// over a cluster with thousands of pods
+	watch.DefaultChanSize = 100000
```

Comment:
should this be configurable? Or is there any downside to defaulting to a larger channel size